AuthenticatorShared/Core/Vault/Repositories/AuthenticatorItemRepositoryTests.swift

```diff
@@ -315,4 +474,37 @@ class AuthenticatorItemRepositoryTests: AuthenticatorTestCase {
 
         XCTAssertEqual(foundItems, [expected.last!])
     }
-}
+
+    // MARK: - Private functions
+
+    private func makeItemListItem(from item: AuthenticatorItem) -> ItemListItem {
```

Comment:
⛏️ This is probably a holdover from the old, old days where functions that had "create" in them [had implications in terms of memory management](https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFDesignConcepts/Articles/NamingConventions.html), but I'm not particularly fond of functions that are `makeSuchAndSuch` (as a synonym of "create". Historically I'd either just make this sort of thing a convenience initializer in an extension, or name it just `itemListItem(from item: AuthenticatorItem)`. It's probably inconsequential (and we probably have counterexamples elsewhere in our codebase), but it's something I noticed.

Reply:
I like `itemListItem(from item: AuthenticatorItem)`. 👍 

This is purely a test convenience method and it involves using fixtures, so I didn't want to put it anywhere that implied it was a normal way of using ItemListItem. It's really just a way to DRY a few things out of the test methods themselves.